publications-2021.bib

```diff
@@ -1,5 +1,19 @@
 % Encoding: US-ASCII
 
+@article{KiRoWi2021,
+	title = {Geometric multigrid with multiplicative Schwarz smoothers for eddy-current and Maxwell's equations in deal.II},
+	journal = {Examples and Counterexamples},
+	volume = {1},
+	pages = {100027},
+	year = {2021},
+	issn = {2666-657X},
+	doi = {https://doi.org/10.1016/j.exco.2021.100027},
```

Comment:
Looking through this again, this isn't actually the DOI. The DOI should look like this:
```suggestion
	doi = {10.1016/j.exco.2021.100027},
```

Reply:
I exported the Bibtex citation from ScienceDirect. That's why I am surprised that the journal returns a wrong Bibtex entry. But I can commit your change to the DOI and try to squash all the commits.